lms/djangoapps/program_enrollments/api/v1/views.py

```diff
@@ -1209,3 +1217,60 @@ def get_course_run_status(course_overview, certificate_info):
             else:
                 return CourseRunProgressStatuses.UPCOMING
         return None
+
+
+class EnrollmentDataResetView(APIView):
```

Comment:
Only Registrar should be hitting this, right? Do we want to make it staff-only?

Reply:
`JWT_RESTRICTED_APPLICATION_OR_USER_ACCESS` is staff only afaik

Reply:
Cool, can you throw in a quick `test_403_for_non_staff` case just to ensure that?